webpack.config.js

```diff
@@ -39,7 +39,6 @@ module.exports = {
           {
             loader: require.resolve('ts-loader'),
             options: {
-              transpileOnly: true,
```

Comment:
should we also run `tsc` on the npm lint for CI?

Reply:
`tsc` is run in the build step of the CI (`build:esm` script), so that would fail in CI if there were issues 👍